src/Lib/Items/CanvasImage.vala

```diff
@@ -131,10 +157,14 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
         manager.get_pixbuf.begin (w, h, (obj, res) => {
             try {
                 var _pixbuf = manager.get_pixbuf.end (res);
+                // TODO: run is_height_small and is_width_small
```

Comment:
The user may resize the image to set the width to 2 for example and the height breaks